RelhaxModpack/RelhaxModpack/RelhaxModpack.csproj

```diff
@@ -533,6 +555,9 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
       <SubType>Designer</SubType>
     </EmbeddedResource>
+    <None Include="..\.editorconfig">
```

Comment:
What's this?

Reply:
A file that vs added to "enforce coding standard in a project", it's been removed.